integration/playwright.config.ts

```diff
@@ -12,9 +12,9 @@ export const common: PlaywrightTestConfig = {
   snapshotDir: './tests/snapshots',
   fullyParallel: true,
   forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 5 : 0,
   timeout: 90_000,
-  maxFailures: process.env.CI ? 1 : undefined,
+  maxFailures: process.env.CI ? 5 : undefined,
```

Comment:
❓  Will this allow tests to pass even if some of them fail? 🤔

Reply:
As I understand the option it means that now in CI 5 tests can fail and only then the rest of the tests is skipped / the suite is stopped: https://playwright.dev/docs/test-parallel#limit-failures-and-fail-fast

So currently we stop the whole suite after one test fails